.travis.yml

```diff
@@ -2,17 +2,27 @@ os: linux
 dist: xenial
 language: node_js
 
+# avoid double Travis build when the PR is created on upstream
+if: |
+  type = pull_request OR \
+  branch = master
```

Comment:
Shouldn't we run builds on `master` to make sure it build properly? Some repositories treat master as unstable, but not sure whether we are doing that for this repo.